crates/re_space_view_time_series/src/space_view_class.rs

```diff
@@ -210,7 +216,7 @@ It can greatly improve performance (and readability) in such situations as it pr
         // For all following lookups, checking indicators is enough, since we know that this is enough to infer visualizability here.
         let Some(indicated_entities) = ctx
             .indicated_entities_per_visualizer
-            .get(&TimeSeriesSystem::identifier())
+            .get(&LegacyTimeSeriesSystem::identifier())
         else {
```

Comment:
Need to patch this: This should take the union of Legacy, Point and Line systems